frontend/src/components/subpages/QuestPageItem.tsx

```diff
@@ -29,7 +31,12 @@ export default function QuestPageItem(props: QuestPageItemProps){
 
     function getLocationsForQuestAndTriggerDirections(quest: Quest){
         getLocationsForQuest(quest.kioskGooglePlacesIds)
-            .then((response: ActiveQuest) => createDirection(response))
+            .then((response: ActiveQuest) => {
+                if(props.dirRenderer?.current){
+                    props.dirRenderer.current.setMap(props.mapRef.current)
+                }
```

Comment:
I think those three lines can be replaced with `props.dirRenderer?.current?.setMap(props.mapRef.current)`